app/navigation/router.js

```diff
@@ -21,6 +21,8 @@ import NavigationModal from './navigation_modal';
 
 const navigationPanResponder = NavigationExperimental.Card.CardStackPanResponder;
 
+const {width: deviceWidth, height: deviceHeight} = Dimensions.get('window');
```

Comment:
This is only used to setup the width initially. I've added this https://github.com/mattermost/mattermost-mobile/pull/344/files#diff-24709eb74a37957ec98095bfbaa6c6e6R194 to handle the device rotation